dev-tools/mage/gotest.go

```diff
@@ -47,7 +47,17 @@ type TestBinaryArgs struct {
 }
 
 func makeGoTestArgs(name string) GoTestArgs {
-	fileName := fmt.Sprintf("build/TEST-go-%s", strings.Replace(strings.ToLower(name), " ", "_", -1))
+	fileName := ""
+	_, isCI := os.LookupEnv("CI")
+	if isCI {
```

Comment:
When running inside CI, the BK plugin will try to link the failure to the step if the generated file has this pattern.

see: https://github.com/buildkite-plugins/junit-annotate-buildkite-plugin#job-uuid-file-pattern-optional